src/eggd_multiqc_to_workbooks.sh

```
@@ -0,0 +1,85 @@
+    echo "Value of path_to_reports_folder: '$path_to_reports_folder'"
+    dx-download-all-inputs --parallel
+    # Fill in your application code here.
+    #
+    # To report any recognized errors in the correct format in
+    # $HOME/job_error.json and exit this script, you can use the
+    # dx-jobutil-report-error utility as follows:
+    #
+    #   dx-jobutil-report-error "My error message"
+
+    export PATH=$PATH:/home/dnanexus/.local/bin
+    echo "$config_file_path"
```

Shellcheck (0.10.0)

[warning] config_file_path is referenced but not assigned.

(SC2154)